web/exercitii/blog/README.md

```diff
@@ -0,0 +1,21 @@
+Propuneți structura HTML pentru blogul lui Tuxy (*45 de minute*)
```

Comment:
Nu trebuie sa adaugi un fisier in acest director. Trebuie respectata structura prezentat la laborator.

Pentru mai multe informatii acceseaza [pagina cursului](https://teaching.alexcoman.com/resurse/tutorial/git/2017/03/11/git-propunerea-unei-solutii/).